src/components/Select/Select/Select.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { useSelect } from 'downshift';
+import { IoMdArrowDropup, IoMdArrowDropdown } from 'react-icons/io';
+import { bem } from '../../../utils/bem/bem';
+import { FieldWrapper } from '../../FieldWrapper';
+import { List } from '../../List';
+import { BasicSelectProps } from '../SelectBase';
+import { SuggestionsList } from '../SuggestionsList';
+import styles from './Select.scss';
+
+const { block, elem } = bem('Select', styles);
+
+interface Props<S> extends Omit<BasicSelectProps<S>, 'isProminent'> {
+    /** The suggestions that is currently selected */
+    selectedSuggestion: S;
+}
+
+export function Select<S>(props: Props<S>) {
```

Comment:
Can we rename the folder containing all of the select components to avoid repetitions? E.g. SelectComponents

Reply:
done